gpytorch/lazy/cat_lazy_tensor.py

```diff
@@ -364,15 +364,32 @@ def devices(self):
     def device_count(self):
         return len(set(self.devices))
 
-    def to(self, device_id):
+    def to(self, *args, **kwargs):
         """
-        returns a new CatLazyTensor with device_id as the output_device
+        Returns a new CatLazyTensor with device as the output_device
```

Comment:
Let's update this and include dtype? And maybe be a bit more clear about how the `*args / **kwargs` are interpreted here?

Reply:
Ah I see you did this below - just copy paste here or get rid of the docstring altogether?